ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -14,47 +14,36 @@ chmod +x ${YQ}
 # Dynamically get CNV catalog image that was provided to the job via gangway API
 CNV_PRERELEASE_CATALOG_IMAGE=$(curl -s https://prow.ci.openshift.org/prowjob?prowjob="${PROW_JOB_ID}" |\
   ${YQ} e '.spec.pod_spec.containers[0].env[] | select(.name == "CNV_PRERELEASE_CATALOG_IMAGE") | .value')
-
 if [ -n "${CNV_PRERELEASE_CATALOG_IMAGE}" ]
 then
   CNV_RELEASE_CHANNEL=stable
-  CNV_SUBSCRIPTION_SOURCE=cnv-prerelease-catalog-source
```

Comment:
This looks like a change in behavior. the CNV_SUBSCRIPTION_SOURCE is not the same as it was previously when the PRERELEASE_VERSION is supplied